test/integration/features/basic.feature

```diff
@@ -0,0 +1,20 @@
+# 
+Feature: Bringup
+  In order to use MiniShift
+  As a user
+  I need to be able to bring up a test environment
+
+  Scenario: Docker environment variables
+    Given a "developer"
+     When executing "minishift start --docker-env=FOO=BAR --docker-env=BAZ=BAT"
+      And should be running
+     Then should have a valid IP address
+     Then executing "minishift ssh echo hello" should contain
+      """
+      hello
+      """
+     Then executing "minishift ssh cat /var/lib/boot2docker/profile" should contain
```

Comment:
Not sure what you mean with the verification of the content of a file, if it has been changed. This  has to be assigned a step in FeatureContext anyways, as there is not distinguishing between 'When', 'Then' or 'And' and no relation can be given to them.

The same goes for the described example. This means that the 'Then' step needs to register an index, based on the command given, because a successive step would need this to know which 'stderr' or 'stdout' is requested. I have no problem with this... but I can not enforce the usage of 'Then' and 'And', they have the same meaning in godog.

A way around this could be:

```
     When executing "minishift ssh cat /var/lib/boot2docker/profile" 
     Then stdout should contain
      """
      export "FOO=BAR"
      export "BAZ=BAT"
      """    
      And stderr should be empty
     When executing "minishift hostfolder mount Users"
     Then stdout should contain:
      """
      ... OK
      """
```
Which means that the index is set by the last executed command (or only the last command is kept on record). This way, no association needs to be kept. WDYT?

It would not be very beneficial to end up in the situation of doing;
```
     Then stdout from "minishift ssh cat /var/lib/boot2docker/profile" should contain
      """
      export "FOO=BAR"
      export "BAZ=BAT"
      """    
      And stderr from "minishift ssh cat /var/lib/boot2docker/profile" should be empty
      And exit status from "minishift ssh cat /var/lib/boot2docker/profile" should be 0
```
as this makes the Gherkin VERY stringent and inflexible.